src/encoder.rs

```diff
@@ -3449,9 +3419,37 @@ pub fn encode_show_existing_frame<T: Pixel>(
   fi: &FrameInvariants<T>, fs: &mut FrameState<T>, inter_cfg: &InterConfig,
 ) -> Vec<u8> {
   debug_assert!(fi.show_existing_frame);
+  let obu_extension = 0;
```

Comment:
What is this constant's purpose?

Reply:
I moved it around when i refactored these functions. It looked superfluous to me as well, but i figured leaving it alone was the better idea, in case verbosity was preferred.

Reply:
I see, I missed its definition in the previous version of the code. I don't have a strong opinion on how to handle it.